Jellyfin.Server/Migrations/Routines/MigrateActivityLogDb.cs

```diff
@@ -75,6 +101,8 @@ public void Perform()
                         newEntry.ItemId = entry[5].ToString();
                     }
 
+                    // Since code references the Id of the entries, this needs to be inserted in order.
+                    // In order to do that, this is needed because EF Core doesn't provide a way to guarantee ordering for bulk inserts.
```

Comment:
```suggestion
                    // In order to do that, we insert one by one because EF Core doesn't provide a way to guarantee ordering for bulk inserts.
```